File: apps/problems/container-with-most-water/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;

    

    //##USER_CODE_HERE##

    public class Main {
      
    public static void main(String[] args) throws IOException {
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      int size_height = Integer.parseInt(br.readLine().trim());
        int[] height = new int[size_height];
        String[] items_height = br.readLine().trim().split("\\s+");
        for (int i = 0; i < size_height; i++) {
          height[i] = Integer.parseInt(items_height[i]);
        }
      Solution obj = new Solution();
      int result = obj.maxArea(height);
      System.out.println(result);
      br.close();
    }
      
    }
  

File: apps/problems/container-with-most-water/boilerplate-full/function.js

    

    //##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/problems/container-with most water/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
const size_height = parseInt(input.shift() || "0");
        const height = input.splice(0, size_height).map(x => parseInt(x));
const result = maxArea(height);
console.log(result);
    